roles/gitlab/README.md

```diff
@@ -0,0 +1,122 @@
+Kayobe Automation Pipeline (GitLab)
+===================================
+
+This Ansible role is capable of generating Gitlab pipeline files for performing CI/CD related activities with OpenStack via Kayobe.
+See the table below for a full list of all the currently supported kayobe automation tasks.
```

Comment:
nit: capitalisation
```suggestion
See the table below for a full list of all the currently supported Kayobe automation tasks.
```

Reply:
There's a few more instances of this in the readme. I won't point them out one by one, since they're easy enough to find with a ctrl+f search